apps/docs/src/content/reference/xr/use-hand-joint.mdx

```diff
@@ -0,0 +1,20 @@
+---
+order: 17
+category: '@threlte/xr'
+name: 'useHandJoint'
+type: 'hook'
+---
+
+Provides a reference to a requested hand joint, once available.
+
+```svelte
+<script>
+  import { useHandJoint } from '@threlte/xr'
+
+  const wristJoint = useHandJoint('left', 'wrist')
+</script>
+```
+
+Reading hand joint positions in real time can be very useful, for example in providing rigid bodies for hands:
+
+<Example path="xr/use-hand-joint" />
```

Comment:
This example is not working on my Quest 2. I see my hands, but the physics interaction is not working, the hands are clipping through the little cubes.

Reply:
This should theoretically be resolved now by [b886469](https://github.com/threlte/threlte/pull/506/commits/b8864697b3e4459e5ba484a54ca4ba706d47020b). I left my headset at my studio, so I can't check, so I'm leaving this open.